src/api/likesAPI.ts

```diff
@@ -18,16 +19,38 @@ export const postProductLikes = async (productId: number) => {
   }
 };
 
-export const deleteProductLikes = async (productId: number) => {
+export async function getProductLikes({ page, size }: GetProductLikesParams): Promise<ProductLike[]> {
   const token = await getCookie('accessToken');
 
   try {
-    await fetch(`${BASE_URL}/api/v1/likes/${productId}`, {
+    const response = await fetch(`${BASE_URL}/api/v1/likes/products?page=${page}&size=${size}`, {
+      cache: 'no-cache',
+      headers: {
+        'Cache-Control': 'no-cache',
+        Authorization: token ? `Bearer ${token}` : '',
+      },
+    });
+    const rawData = await response.json();
+
+    return rawData.data;
+  } catch (error) {
+    throw error;
+  }
+}
+
+export const deleteProductLikes = async (productId: number[]) => {
+  const token = await getCookie('accessToken');
+
+  try {
+    await fetch(`${BASE_URL}/api/v1/likes`, {
       method: 'DELETE',
       headers: {
         'Content-Type': 'application/json',
         Authorization: `Bearer ${token}`,
       },
+      body: JSON.stringify({
+        productIds: productId,
```

Comment:
parameter를 productIds로 바꾸면 코드 더 줄일 수 있을 것 같아요
```suggestion
        productId
```